src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.ToJson.PipeWriter.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+#if !BUILDING_INBOX_LIBRARY
+        [System.CLSCompliantAttribute(false)]
```

Comment:
Is this causing a warning? Maybea dd a no warn exception for now (`<NoWarn>3021</NoWarn>`) since I don't think it will be needed within the source once the code is in corefx (it will only be required in the ref assembly).

Reply:
Yes this is temporary for corefxlab.